### PR TITLE
feat: add github issue templates

### PR DESCRIPTION
**Pull request by richardeschloss**
Rebased onto template branch. Merging request to base template branch.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+```
+-->
+
+### Reproduction Link
+<!-- A minimal test case on https://template.nuxtjs.org/ or GitHub reprository that can reproduce the bug. -->
+
+### Steps to reproduce
+
+
+### What is Expected?
```
Comment (manniL) [inline .github/ISSUE_TEMPLATE/bug_report.md:33]:
```suggestion
### What is expected?
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+### Nuxt configuration for {{ name }}
+<!--
+    If relevant, please include the configuration you are using for this module.
+    For example:
+```
+```
+-->
+
+### Reproduction Link
```
Comment (manniL) [inline .github/ISSUE_TEMPLATE/bug_report.md:27]:
```suggestion
### Reproduction link
```

**Comment by Atinux** [comment]
Looks good to me, asking module authors to see if they see some improvements :)

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+<!-- A minimal test case on https://template.nuxtjs.org/ or GitHub reprository that can reproduce the bug. -->
```
Comment (manniL) [inline .github/ISSUE_TEMPLATE/bug_report.md:28]:
```suggestion
<!-- A minimal test case on https://template.nuxtjs.org/ or GitHub reprository that can reproduce the bug. -->
```

Maybe it'd be good to:

* state that a reproduction is very important
* jsfiddle/jsbin/codepen is no proper reproduction

Have seen these issues mostly regarding reproduction links/repos

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+title: ''
+labels: enhancement
+assignees: ''
+
+---
+
+### Is your feature request related to a problem? Please describe.
+<!-- A clear and concise description of what the problem is. Ex. I'm always frustrated when [...] -->
+
+### Describe the solution you'd like
```
Comment (manniL) [inline .github/ISSUE_TEMPLATE/feature_request.md:13]:
```suggestion
### Describe the solution you'd like to see
```

File: .github/ISSUE_TEMPLATE/question.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
+---
+name: Question
```
Comment (manniL) [inline .github/ISSUE_TEMPLATE/question.md:2]: Do we want to have questions inside the GH repo instead of bug reports & feature requests only?

**Comment by Atinux** [comment]
I just saw that the issue templates are in the wrong repo, @richardeschloss could you move them into `template/.github` like you did in #49

File: .github/ISSUE_TEMPLATE/question.md (hunk 5 above)
Comment (Atinux) [reply .github/ISSUE_TEMPLATE/question.md:2]: I think for modules it's good enough to start, they can always remove it or change it later on

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 3 above)
Comment (Atinux) [reply .github/ISSUE_TEMPLATE/bug_report.md:28]: Sounds good, could you update it please?

Comment (richardeschloss) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Ok, I just updated the bug report.

**Comment by Atinux** [comment]
Actually, the branch to merge should be `master`, we don't use GitHub template system since ours gives more control.

**Comment by richardeschloss** [comment]
Ok. I can move it.

**Comment by pimlie** [comment]
I feel eg the bug template a bit too verbose, something like this looks a bit cleaner:

---
name: Bug report
about: Report a bug report to help us improve the module.
title: ''
labels: bug
assignees: ''

---

## Reproduction
> :warning: without a minimal reproduction we wont be able to look into your issue

**Link:** https:///codesandbox.io/

#### What is expected?

#### What is actually happening?

#### Steps to reproduce

## Additional information


## Checklist
- [ ] I have tested with the latest Nuxt version and the issue still occurs
- [ ] I have tested with the latest {{ name }} version and the issue still occurs
- [ ] I have added a minimal reproduction that clearly demonstrates my issue
- [ ] I have searched the issue tracker and this issue hasnt been reported yet

**Comment by Atinux** [comment]
Dammit forget my comment @ricardogobbosouza we can keep using `template` branch after talking with @pi0

**Comment by richardeschloss** [comment]
ok, no worries. either way, both solutions are available now. my "template" branch goes to nuxt's "template". my "feat/issue_template" branch would go to nuxt's master

**Comment by Atinux** [comment]
I like your proposition @pimlie, just make sure to use https://template.nuxtjs.org for codesandbox link

Also since we cannot have interpolation (`{{ x }}`) in issue template, better to make it generic (e.g: this module).

**Comment by richardeschloss** [comment]
Just to confirm...I think I captured the ideas discussed here: We can leave the module name generic as "module" because it's obvious the user is on the page...they just look up 2 cm, they know the module name. 

---
name: Bug report
about: Report a bug report to help us improve the module.
title: ''
labels: bug
assignees: ''

---

### Version
module: 
nuxt: 

### Nuxt configuration
#### [mode](https://nuxtjs.org/api/configuration-mode): 
 - [ ] universal
 - [ ] spa

### Nuxt configuration


## Reproduction
> :warning: without a minimal reproduction we wont be able to look into your issue

**Link:** 
[ ] https:///codesandbox.io/
[ ] GitHub repository

#### What is expected?
#### What is actually happening?
#### Steps to reproduce
## Additional information
## Checklist
* [ ]  I have tested with the latest Nuxt version and the issue still occurs
* [ ]  I have tested with the latest module version and the issue still occurs
* [ ]  I have searched the issue tracker and this issue hasn't been reported yet

### Steps to reproduce


### What is expected?


### What is actually happening?


### Performance analysis?

File: .github/ISSUE_TEMPLATE/config.yml [hunk 6]
```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Nuxt Community Discord
+    url: https://discord.nuxtjs.org/
+    about: Consider asking questions about the module here.
+  - name: {{ name }} documentation
```
Comment (pi0) [inline .github/ISSUE_TEMPLATE/config.yml:6]: Can we maybe comment out documentation lines? Templates are not supported and modules usually only have a README as docs unless one creates, which as a chore task, can also enable/add this link

**Comment by pi0** [comment]
Rest seems awesome thanks @ricardogobbosouza :green_heart:

File: .github/ISSUE_TEMPLATE/config.yml (hunk 6 above)
Comment (richardeschloss) [reply .github/ISSUE_TEMPLATE/config.yml:6]: Ok, I agree.

Comment (richardeschloss) [reply .github/ISSUE_TEMPLATE/config.yml]: Addressed. I commented out the docs section, but left in a "/README.md" as a default (commented out).

**Comment by ricardogobbosouza** [comment]
Very nice @richardeschloss

**Pull request merged**